### PR TITLE
#578 - implement choose default address for user profile

### PR DESCRIPTION
**Pull request by kiettrangthuan**

### CODE REVIEW

File: customer/src/main/java/com/yas/customer/viewmodel/AddressGetListResponseVm.java [hunk 1]
```diff
@@ -0,0 +1,4 @@
+package com.yas.customer.viewmodel;
+
+public record AddressGetListResponseVm(Long id, Boolean isActive) {
```
Comment (LEHOANGGLAM) [inline customer/src/main/java/com/yas/customer/viewmodel/AddressGetListResponseVm.java:3]: confuse name, this is not get List, refer AddressGetVm or something else.

File: storefront/modules/address/models/AddressGetListModel.ts [hunk 2]
```diff
@@ -0,0 +1,4 @@
+export type AddressGetListModel = {
```
Comment (LEHOANGGLAM) [inline storefront/modules/address/models/AddressGetListModel.ts:1]: same here

**Review by LEHOANGGLAM — APPROVED** [review]
Dont forget to change a suitable name for Vm

File: customer/src/main/java/com/yas/customer/controller/UserAddressController.java [hunk 3]
```diff
@@ -34,4 +35,10 @@ public ResponseEntity deleteAddress(@PathVariable Long id) {
         userAddressService.deleteAddress(id);
         return ResponseEntity.ok().build();
     }
+
+    @PutMapping("/storefront/user-address/{id}")
+    public ResponseEntity chooseDefaultAddress(@PathVariable Long id) {
+        userAddressService.chooseDefaultAddress(id);
+        return ResponseEntity.ok().build();
```
Comment (khoahd7621) [inline customer/src/main/java/com/yas/customer/controller/UserAddressController.java:42]: return type should be ResponseEntity<Void>

return statement should be return ResponseEntity.noContent().build();

**Review by khoahd7621 — APPROVED** [review]
Reviewed, change if you think it necessary.

**Pull request merged**